pangloss/__init__.py

```diff
@@ -1,3 +1,27 @@
+"""
+Line of sight mass reconstruction in the Universe.
+"""
+
+# These are the static paths for data.
+# They are fetched via data_fetcher.py
+
+from os import environ
+
+PANGLOSS_DIR = environ['PANGLOSS_DIR']
```

Comment:
Can you just double check that the README explains that this environment var is set, please? And then the README should now also show how the test data can be obtained. It might be best to simply have the README point to some the list of notebooks, saying "to see Pangloss at work on some test data..." or somesuch. What do you think?